common/test/acceptance/pages/studio/utils.py

```diff
@@ -247,10 +247,13 @@ def verify_ordering(test_class, page, expected_orderings):
 
 def click_studio_help(page):
     """Click the Studio help link in the page footer."""
+    EmptyPromise(lambda: page.q(css='.cta-show-sock').visible, "Help link visible").fulfill()
+
     page.q(css='.cta-show-sock').click()
+
     EmptyPromise(
         lambda: page.q(css='.support .list-actions a').results[0].text != '',
-        'Support section opened'
+        'Support section opened', timeout=60
```

Comment:
@muzaffaryousaf Can you check if test works without this timeout as now we have promise in the start of this method.